### PR TITLE
[13.0][ADD] purchase_substate

### PR DESCRIPTION
**Pull request by kittiu**
Depends on
- [x] base_substate https://github.com/OCA/server-ux/pull/199

### CODE REVIEW

**Review by ps-tubtim — APPROVED** [review]

File: purchase_substate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{
+    "name": "Purchase Sub State",
+    "version": "13.0.1.0.0",
+    "category": "Tools",
+    "author": "Akretion,Ecosoft,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/purchase-workflow/",
+    "license": "AGPL-3",
+    "depends": ["base_substate", "purchase"],
+    "data": ["views/purchase_views.xml",
+             'data/purchase_substate_mail_template_data.xml',
```
Comment (newtratip) [inline purchase_substate/__manifest__.py]:
```suggestion
             "data/purchase_substate_mail_template_data.xml",
```

File: purchase_substate/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+             'data/purchase_substate_data.xml',
```
Comment (newtratip) [inline purchase_substate/__manifest__.py]:
```suggestion
             "data/purchase_substate_data.xml",
```

File: purchase_substate/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+             ],
+    "demo": [
+        'data/purchase_substate_demo.xml',
```
Comment (newtratip) [inline purchase_substate/__manifest__.py]:
```suggestion
        "data/purchase_substate_demo.xml",
```

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Pull request merged**